frontend/packages/console-app/src/components/user-preferences/language/getLastLanguage.ts

```diff
@@ -1,3 +1,4 @@
 import { LAST_LANGUAGE_LOCAL_STORAGE_KEY } from './const';
 
-export const getLastLanguage = (): string => localStorage.getItem(LAST_LANGUAGE_LOCAL_STORAGE_KEY);
+export const getLastLanguage = (): string =>
+  localStorage.getItem(LAST_LANGUAGE_LOCAL_STORAGE_KEY) || navigator.language;
```

Comment:
we should be checking the `languages` array
```suggestion
  localStorage.getItem(LAST_LANGUAGE_LOCAL_STORAGE_KEY) || navigator.languages[0] || navigator.language;
```

@rawagner could you elaborate?

Reply:
> navigator.languages

@jhadvig What we need is the user browser preferred lng when the `localStorage` is cleared by the "Use the default browser language setting." action in user preference > lng tab. The `navigator.language` returns the lng while `navigator.languages` an array of lngs with index 0 equal to `navigator.language`. So I think it is better and clearer to use ``navigator.language` except if you know of any reason not to. Both will always return the same value.